Dockerfile

```diff
@@ -22,12 +20,14 @@ RUN apt-get update && \
     apt-get clean
 
 WORKDIR $GOPATH/src/udr
+
 COPY . .
 RUN make all
 
 FROM alpine:3.20 AS udr
 
-LABEL description="ONF open source 5G Core Network" \
+LABEL maintainer="Aether SD-Core <dev@aetherproject.org>" \
```

Comment:
in some other changes you have dev@list.aetherproject.org
Could you pls check which one is correct?

Reply:
Done. It should be: dev@lists.aetherproject.org